sdk/config.go

```diff
@@ -76,6 +80,12 @@ func getStringConfig(envVar string, defaultValue string, config attribute.KeyVal
 	return config.Value.AsString()
 }
 
+func isCollectorInsecure(attrs []attribute.KeyValue) bool {
+	collectorInsecureConfig := getConfigByKey(collectorInsecureKey, attrs)
+	bool, _ := strconv.ParseBool(getStringConfig(collectorInsecureEnvVar, defaultCollectorInsecure, collectorInsecureConfig))
```

Comment:
What if the parsing fails? Let's handle this case - simply by returning false

Reply:
done.